### ISSUE TITLE
support 0.10.x

### ISSUE THREAD

**Issue by kirrg001** [thorn0/timezoned-date#3]
I already saw your branch (https://github.com/thorn0/timezoned-date/tree/node10x) for fixing the issue with ```setPrototypeOf```.

Their might be another issue with ```Symbol```.
https://phabricator.babeljs.io/T2745

**Comment by kirrg001** [comment]
And after fixing ```Symbol``` and ```Object.setPrototypeOf```, 7 of 9 tests passes. 

I'm getting a TypeError for 2 of 9 tests:

```
moment('2016-01-02 08:00:00').format('YYYY-MM-DD HH:mm:ss').should.eql('2016-01-02 08:00:00');

TypeError
      at Date.<anonymous> (node_modules/timezoned-date/lib/index.js:173:23)
```

**Comment by thorn0** [comment]
Yep, Travis has finally checked my commit:
https://travis-ci.org/thorn0/timezoned-date/jobs/132335478

**Comment by thorn0** [comment]
fixed in 2.0.15 (https://github.com/thorn0/timezoned-date/commit/63f9cba87e00538cfb9ba05f9b19e376dccceb30)
